br/pkg/restore/config.go

```diff
@@ -0,0 +1,10 @@
+package restore
+
+import "github.com/pingcap/tidb/parser/model"
+
+var incrementalRestoreActionBlacklist = []model.ActionType{
```

Comment:
if this file only contains this variable, I think you can put this variable and function in same file.